src/formatDate.js

```diff
@@ -8,7 +8,28 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat[3];
+  const separatorTo = toFormat[3];
+
+  const dateParts = date.split(separatorFrom);
+  const dateObject = {};
+
+  fromFormat.slice(0, 3).forEach((part, index) => {
+    dateObject[part] = dateParts[index];
+  });
+
+  if (dateObject['YYYY']) {
+    dateObject['YY'] = dateObject['YYYY'].slice(-2);
+  } else if (dateObject['YY']) {
+    const year = parseInt(dateObject['YY'], 10);
+
+    dateObject['YYYY'] =
+      year < 30 ? '20' + dateObject['YY'] : '19' + dateObject['YY'];
```

Comment:
When converting from 'YY' to 'YYYY', it's a good practice to ensure that the year prefix ('19' or '20') is not hard-coded. Consider defining these as constants at the beginning of the function to make it easier to change in the future if needed.

Reply:
Consider fixing the previous review comments